# HG changeset patch
# User marsgui
# Date 1476384175 -7200
#      Thu Oct 13 20:42:55 2016 +0200
# Node ID 4f030d62420c3101d85976f5017b29070d1277b4
# Parent  58ff524b7f0f0b8127333ba9542c9619dbdd4dbf
Fix Debian bug #840189: drop multirow dependency

diff -r 58ff524b7f0f -r 4f030d62420c latex/style/dbk_table.sty
--- a/latex/style/dbk_table.sty	Sun Oct 02 23:21:03 2016 +0200
+++ b/latex/style/dbk_table.sty	Thu Oct 13 20:42:55 2016 +0200
@@ -9,7 +9,6 @@
 \usepackage{longtable}
 \usepackage{lscape}
 \usepackage{colortbl}
-\usepackage{multirow}
 \usepackage{calc}
 \usepackage{hhline}
 
@@ -21,16 +20,6 @@
   \expandafter[\expandafter3\expandafter]\expandafter{%
   \multicolumn{#1}{#2}{#3}}
 
-% Make \@xmultirow long
-\expandafter\long\expandafter\def%
-\expandafter\@xmultirow%
-\expandafter#\expandafter1%
-\expandafter[\expandafter#\expandafter2\expandafter]%
-\expandafter#\expandafter3%
-\expandafter[\expandafter#\expandafter4\expandafter]%
-\expandafter#\expandafter5%
-\expandafter{\@xmultirow{#1}[#2]{#3}[#4]{#5}}
-
 % For the newtbl code
 \newdimen\newtblstarfactor%
 \newdimen\newtblsparewidth%
